### ISSUE TITLE
Rule Change: Highlight fences only in fenced-code-language rule reports

### ISSUE THREAD

**Issue by TKDev7** [eslint/markdown#476]
### What rule do you want to change?

fenced-code-language

### What change do you want to make?

Highlight fences only in fenced-code-language rule report

### How do you think the change should be implemented?

Other

### Example code

\

\```js
const message = "Hello, world!";
console.log(message);
\```

### What does the rule currently do for this code?

Currently, when a violation occurs (e.g., missing language or disallowed language), the rule reports the entire code block.

### What will the rule do after it's changed?

After this change, the rule will only highlight the relevant parts of the code block for a violation.
- For a **missing language** error, it will highlight only the **opening fences**.
- For a **disallowed language** error, it will highlight the **opening fence and the specified language**.

### Participation

- [x] I am willing to submit a pull request to implement this change.

### Additional comments

This change will make the reported locations more precise and similar to how markdownlint highlight issues, making it easier for developers to pinpoint the exact source of the problem without distracting from the entire code content.

**Comment by nzakas** [comment]
This seems like a good idea to me. 👍 Marking as accepted.
